### ISSUE TITLE
'NoneType' is not iterable: error during training

### ISSUE THREAD

**Issue by Kervin-L** [dBeker/Faster-RCNN-TensorFlow-Python3#111]
Hello,I run train.py error.The log is 
Traceback (most recent call last):
  File "train.py", line 216, in <module>
    train.train()
  File "train.py", line 123, in train
    variables_to_restore = self.net.get_variables_to_restore(variables, var_keep_dic)
  File "D:\Project\MyLearning\Faster-RCNN-TensorFlow-Python3-master\lib\nets\vgg16.py", line 66, in get_variables_to_restore
    if v.name.split(':')[0] in var_keep_dic:
TypeError: argument of type 'NoneType' is not iterable

Thank you!

**Comment by morpheusthewhite** [comment]
Duplicate of https://github.com/dBeker/Faster-RCNN-TensorFlow-Python3/issues/80

**Comment by Kervin-L** [comment]
@morpheusthewhite Hi,I run python demo.py error.
The log:
Traceback (most recent call last):
  File "demo.py", line 142, in <module>
    raise NotImplementedError
NotImplementedError

**Comment by morpheusthewhite** [comment]
Are you passing any argument to `demo`?

**Comment by Kervin-L** [comment]
No,I want to  run demo.py to test environment.But it report an error.

**Comment by morpheusthewhite** [comment]
you can run `demo` only once you've finished training

**Comment by Kervin-L** [comment]
@morpheusthewhite Yes,I have run train.py successfully,and I use the trained model.

**Comment by morpheusthewhite** [comment]
Then I repeat the same question

> Are you passing any argument to `demo`?

I mean, do you just run `python demo.py`?

**Comment by Kervin-L** [comment]
@morpheusthewhite Yes,I only python demo.py.Could you tell me how to run the demo.py successfully?

**Comment by liu978079474** [comment]
@morpheusthewhite After running python train.py,I transfer the trained model in demo.py. But when I ran python demo.py, I got an error.
Traceback (most recent call last):
File "demo.py", line 142, in
raise NotImplementedError
NotImplementedError

**Comment by morpheusthewhite** [comment]
Post here the content of your folders (if you're on Linux just run `tree -L 3`).

And stop posting the same question
